modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -64,8 +64,19 @@ def fetch_errored(claim)
 
       def format_evss_errors(errors)
         errors.map do |error|
-          formatted = error['key'] ? error['key'].gsub('.', '/') : error['key']
-          { status: 422, detail: "#{error['severity']} #{error['detail'] || error['text']}".squish, source: formatted }
+          if error&.dig('text')
+            formatted = error['key']
+            { status: 422, detail: "#{error['severity']} || #{error['text']}".squish,
+              source: formatted }
+          elsif error&.dig('text', 'messages')
+            formatted = error['text']['messages'][0]['key']
```

Comment:
I _think_ this will also need the `.gsub('.', '/')` to keep the returned source value the same